recipe/meta.yaml

```
@@ -34,7 +34,7 @@ requirements:
     - scikit-learn
     - scipy
     - tqdm
-    - pytorch
+    - pytorch {{ pytorch_version }}
```

```suggestion
    - pytorch 1.10
```

I think we made a mistake a pinned to tightly. I would suggest loosening this one